curriculum/js2/6.js

```diff
@@ -0,0 +1,12 @@
+/**
+*  * Given an array, return an array of numbers larger than 5
+  *   * @param {array} a
+  *    * @returns {array}
+  *     */
+const solution = (a) => {
+  return  a.filter((element, index) => a[index] > 5)
```

Comment:
if you're going to use the filter function might as well right it like this:
` a.filter((element) => element > 5)`
but this defeats the purpose of writing the problem recursively. So can you please re-write this problem the recursive way without a filter method?